Mathlib/CategoryTheory/Limits/Preserves/Shapes/Kernels.lean

```diff
@@ -229,4 +286,36 @@ theorem preserves_cokernel_iso_comp_cokernel_map {X' Y' : C} (g : X' ⟶ Y') [Ha
 
 end Cokernels
 
+variable (X Y : C) (G : C ⥤ D) [Functor.PreservesZeroMorphisms G]
```

Comment:
```suggestion
variable {X Y : C} (G : C ⥤ D) [Functor.PreservesZeroMorphisms G]
```

Reply:
I am not sure it is better to use `{X Y : C}` here. If the instance `preservesKernelZero` is found automatically, it does not matter very much. Otherwise, if we need to create the corresponding term of type `PreservesLimit (parallelPair (0 : X ⟶ Y) 0) G`, it is more convenient to use `(X Y : C)` so that we can write `preservesKernelZero X Y G`.

Reply:
Yes, good point.